pkg/config/admin_secrets_test.go

```diff
@@ -14,15 +15,17 @@ func TestAdminSecrets(t *testing.T) {
 
 	testFile, err := ioutil.ReadFile("admin_secrets_test.yaml")
 	assert.NoError(t, err)
+	secretsFromFile := config.AdminSecretsConfig{}
+	err = yaml.Unmarshal(testFile, &secretsFromFile)
 
 	service := config.AdminSecretsService{}
 	service.Flags.DefaultAdminPassword = "mysecret"
 	err = service.NewAdminSecretsConfig()
 	assert.NoError(t, err)
 
-	s, err := service.Secrets.String()
-	tests.Debugf("%s", s)
+	secretsFromService := service.Secrets
+	tests.Debugf("%s", secretsFromService)
 	assert.NoError(t, err)
```

Comment:
err won't have changed as you are no longer updating it as you removed the line

Reply:
true.